css/.csscomb.json

```diff
@@ -13,11 +13,9 @@
     "combinator-space": [" ", " "],
     "element-case": "lower",
     "eof-newline": true,
-    "leading-zero": null,
```

Comment:
Yup, they are both disabled. From the [rules doc](https://github.com/sasstools/sass-lint#rules):

> For all rules, setting their severity to 0 turns it off